File: elias/Problema0434/problema0434.js
const prompt = require("prompt-sync")({ sigint: true })

function calculaResultado(A, B, C){
    var soma = A + B + C;
    var resultado = soma * soma;
    return resultado;
}
var A = parseInt(prompt("Digite um número inteiro para A:"));
var B = parseInt(prompt("Digite um número inteiro para B:"));
var C = parseInt(prompt("Digite um número inteiro para C:"));
var resultado = calculaResultado(A, B, C);
console.log("O quadrado da soma dos três valores é: "+resultado);

File: elias/Problema0434/problema0434.java
import java.util.Scanner;

public class problema0434 {

    public static int calculaResultado(int A, int B, int C){
        int soma = A + B + C;
        int resultado = soma * soma;
        return resultado;
    }

    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int A, B, C;
        System.out.println("Digite um número inteiro para A:");
        A = Integer.valueOf(input.nextLine());
        System.out.println("Digite um número inteiro para B:");
        B = Integer.valueOf(input.nextLine());
        System.out.println("Digite um número inteiro para C:");
        C = Integer.valueOf(input.nextLine());
        int resultado = calculaResultado(A, B, C);
        System.out.println("O quadrado da soma dos três valores é: "+resultado);
    }
}